### ISSUE TITLE
Why use FullConnectCriterion to pretrain the transition?

### ISSUE THREAD

**Issue by chbrian** [facebookresearch/wav2letter#40]
I noticed in the code, before training the network and asgcriterion, the transition is trained by FullConnectCriterion as a kind of pretraining.
What's the intuition behind this? Thank you.

**Comment by andresy** [comment]
We use the LinearSegCriterion to *optionally* pre-train ASG transitions; it is a forced-align training: given a sequence and a number of labels, it assigns label linearly along the sequence.

**Comment by chbrian** [comment]
Thank you @andresy .
